utils.py

```diff
@@ -0,0 +1,551 @@
+"""Lifetimes utils and helpers."""

```

Comment:
hm, did you mean to edit the lifetimes.utils python file? As is, this PR adds a brand new `utils.py` file.

Reply:
That's what you get for being lazy and using the web interface rather than a proper client -.-
This is fixed, now though there seem to be some some residual issues with period values which I will fix asap.